app/src/main/java/ru/vssemikoz/newsfeed/newsfeed/NewsFeedContract.java

```diff
@@ -54,5 +56,7 @@
 
         // TODO: 26.03.2020 think about better solution
         void setView(NewsFeedContract.View view);
+
+        void shareNews(int position);
```

Comment:
Looks like you will share a single item rather than a list of news We need to rename this method.